src/Diggin/Bridge/Guzzle/AutoCharsetEncodingPlugin/AutoCharsetEncodingPlugin.php

```diff
@@ -13,6 +13,17 @@ class AutoCharsetEncodingPlugin implements EventSubscriberInterface
      * @var DocumentConverter
      */
     protected $charset_front;
+    /**
+     * @var string
+     */
+    private $contentTypes;
+
+    /**
+     * @param array $contentTypes
+     */
+    public function __construct($contentTypes = array('text/html')) {
+        $this->contentTypes = '[' . implode('|', $contentTypes) . ']';
```

Comment:
Could you add checking $contentTypes is not empty ? (If empty, throw Exception )